tla/consensus/SIMccfraft.cfg

```diff
@@ -72,8 +72,9 @@ INVARIANTS
     CandidateTermNotInLogInv
     ElectionSafetyInv
     LogMatchingInv
-    QuorumLogInv
-    LeaderCompletenessInv
+    \* Disabled until retirement is modeled correctly in the spec
```

Comment:
Leaving this here as an alternative to disabling invariants completely if it is possible to find a predicate `P` that equals false when the invariant is expected not to hold.  Then, in ccfraft:

```tla
QuorumLogInv == P => OriginalQuorumLogInv
```

Reply:
It is likely that the QuorumLogInv can and should be reinstated with a change to account for retired nodes.